widgets/DynamicSymbology/Widget.js

```diff
@@ -292,6 +268,9 @@ function(declare, BaseWidget, LayerInfos, dom, domConstruct, on, Map, Color, Col
 
 			geoenrichedFeatureLayer.setRenderer(gRenderer);
 			geoenrichedFeatureLayer.redraw();
+            
+            geoenrichedFeatureLayer.setVisibility(false);
+            geoenrichedFeatureLayer.setVisibility(true);
 
```

Comment:
Interesting - so just toggling the visibility is sufficient to trigger the legend to redraw? Nice easy solution. Odd that Esri wouldn't have some other trigger, but what the heck?